Project.toml

```diff
@@ -19,7 +19,7 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 [compat]
 CEnum = "0.4, 0.5"
 ChainRulesCore = "1"
-EnzymeCore = "0.7.3"
+EnzymeCore = "0.7.3, 0.7.4"
```

Comment:
This needs to be `0.7.4` since otherwise you will get a "EnzymeCore.make_zero!" not defined.

Reply:
i was just wondering this